app/src/main/java/org/p2p/wallet/auth/ui/generalerror/timer/OnboardingGeneralErrorTimerFragment.kt

```diff
@@ -17,36 +18,51 @@ import org.p2p.wallet.utils.withArgs
 import org.p2p.wallet.auth.ui.generalerror.timer.OnboardingGeneralErrorTimerContract.View as ContractView
 
 private const val ARG_TIMER_ERROR_TYPE = "ARG_TIMER_ERROR_TYPE"
+private const val ARG_PHONE_NUMBER = "ARG_PHONE_NUMBER"
```

Comment:
we usually use EXTRA prefix to consts

Reply:
Found ARG prefix in onboarding screens. Looks better for `argument` values. Guys, what do you think?

Reply:
<img width="931" alt="Screen Shot 2022-08-29 at 12 29 11" src="https://user-images.githubusercontent.com/100791035/187170357-133168c9-c6ce-49c5-9bee-6894d2c6e0bf.png">